api.planx.uk/saveAndReturn/utils.ts

```diff
@@ -296,11 +296,9 @@ const markSessionAsSubmitted = async (sessionId: string) => {
  * Get formatted list of the session's project types
  */
 const getHumanReadableProjectType = async (
```

Comment:
This function is also required in the frontend, so I'll move this (in this form) to `planx-core` in another PR.

Reply:
This is related to a small change I opened here: https://github.com/theopensystemslab/planx-new/pull/1626

We made slightly different decisions in refactoring: I updated the argument to be a passport-like dict, whereas you've gone for the list of passport variable strings. 

In my mind, a small benefit to the first is then that we have fewer instances of the actual passport variable `proposal.projectType` to manage & update if it continues to live directly in the function rather than at the call-level.

But not strongly tied to one way or the other. Just flagging as an expected merge conflict.

Reply:
I think your instinct is correct here - a function related to project types is best responsible for knowing which key to fetch as opposed to multiple callers 👍 We'll merge your PR, then I'll rebase and resolve conflicts.